site/entry.js

```diff
@@ -59,29 +59,24 @@ class Entry extends React.Component {
       }
     }
 
+    const selectorElClasses = `mr3 pr3 px3 color-blue-dark round bg-blue-faint mb3 inline-block ${
+      selectors.length > 1 ? 'txt-s' : ''
+    } ${selectors.length > 20 ? 'txt-xs' : ''}`;
```

Comment:
The jump in size feels pretty weird here. Why not just stick to `txt-s` as overflow provides scrolling?